### DISCUSSION TITLE
Idea for a desktop app

### DISCUSSION THREAD

**Discussion by mhurk** [mhurk/marathon#13] [Ideas]
Very first version. Marathon parameters (# runners, # per minute start, waves or no waves, time between full and half, pace per distance and perhaps more) need to be added, and default proposed.

This could be a browser app too but that adds work for setting up a server.

![image](https://github.com/user-attachments/assets/88ab5c9f-6ae3-48b1-9536-5d0cd8852045)

**Comment by rwestein** [comment]
If you want other to use it, it's probably better to have a browser app, because that avoids an installation process to use it (at ASML, that's always difficult)

**Reply by mhurk** [reply]
True, but that has the hassle (and costs) for a server. I was thinking of an executable that does not require installation. So not full blown with start menu (start menu probably does not need admin rights) and integration. Web is much easier and more flexible to update and change.
